templates/base/footer_content.tmpl

```diff
@@ -2,6 +2,9 @@
 	<div class="ui container">
 		<div class="ui left">
 			© Gitea {{if (or .ShowFooterVersion .PageIsAdmin)}}{{.i18n.Tr "version"}}: {{AppVer}}{{end}} {{if ShowFooterTemplateLoadTime}}{{.i18n.Tr "page"}}: <strong>{{LoadTimes .PageStartTime}}</strong> {{.i18n.Tr "template"}}: <strong>{{call .TmplLoadTimes}}</strong>{{end}}
+			<span class="ui right links">
+				{{template "custom/extra_links_footer" .}}
```

Comment:
I think this should be moved between line 9 and 10 below

Reply:
@lafriks I encapsulate the template inside `span` so that links can be put beside gitea version

Reply:
Imho better would be to add them between home page link and go version string